### ISSUE TITLE
inverse_laplace_transform problems in sympy 1.12 

### ISSUE THREAD

**Issue by SIMPHYSICLAB** [sympy/sympy#25796]
I am having problems with sympy.inverse_laplace_transform in version 1.12 of sympy, the expected result for the function

```
TF=(5 / (s**3 + 4.5*s**2 + 6.5*s + 8) )*(1/s)
```
is:
```
y(t)=0.625−0.1699exp(−3.259t)−0.5812exp(−0.621t)sin(1.439t)−0.4551exp(−0.621t)cos(1.439t)
```
my result is:

```
y(t)= 1.25−0.169935589280414e
−3.25869675702641t
 −0.581236260403932e
−0.620651621486795t
 sin(1.43866629639448t)−0.455064410719586e
−0.620651621486795t
 cos(1.43866629639448t)
```

the result is the same but the constant changes and therefore the value in permanent regime is not the same

I have tried previous versions but they do not work well in many cases (even removing the decimals), 1.12 is the most stable.

**Comment by SIMPHYSICLAB** [comment]
Other versions do it well in this case  giving the result y(t)=0.625−0.1699exp(−3.259t)−0.5812exp(−0.621t)sin(1.439t)−0.4551exp(−0.621t)cos(1.439t), but as I say, many other cases give an error directly.

**Comment by hanspi42** [comment]
Hi all, thanks for reporting this! It is already solved in the development version. As it happens, I will give a lesson about python development at a secondary school next week. This is a wonderful example for how bug reporting in open-source development works and how the developers then find out whether the bug was fixed knowingly or by chance (spoiler alert: it was fixed knowingly). I will show the debugging process and reply to and close this issue during that lesson, so please leave it opnen.

**Comment by hanspi42** [comment]
Thank you for your patience ;)

This has already been fixed in PR #25642, where we solved issue #25352.
